templates/controllers/grid/queries/form/queryForm.tpl

```diff
@@ -61,6 +61,10 @@
 
 		<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
 
+		{if $allowedEditTimeNotice['show']}
+			<p><span class="sub_label">{translate key="submission.query.allowedEditTime" allowedEditTimeNoticeLimit="{$allowedEditTimeNotice['limit']}"}</span></p>
```

Comment:
`allowedEditTimeNoticeLimit="{$allowedEditTimeNotice['limit']}"}` could be simplified to `allowedEditTimeNoticeLimit=$allowedEditTimeNotice['limit']`